addons/product/models/product_template.py

```diff
@@ -22,7 +22,7 @@ class ProductTemplate(models.Model):
     @tools.ormcache()
     def _get_default_category_id(self):
         # Deletion forbidden (at least through unlink)
-        return self.env.ref('product.product_category_all')
+        return self.env.ref('product.product_category_5')
```

Comment:
Previously, the default category was a data, now it's a demo data, so not always installed. Sound like a bad idea IMO